src/dmd/builtin.d

```diff
@@ -475,6 +475,17 @@ public extern (C++) void builtin_init()
         add_builtin("_D4core5bitop7_popcntFNaNbNiNfmZi", &eval_popcnt);
 }
 
+/**
+ * Deinitializes the global state of the compiler.
+ *
+ * This can be used to restore the state set by `builtin_init` to its original
+ * state.
+ */
+public void builtinDeinitialize()
```

Comment:
- Why not `deinitialize` like in the other modules?
- Why not `static` like in the other modules?

Reply:
> Why not deinitialize like in the other modules?

To better match the existing `builtin_init`. At the call site having something called just `deinitialize` is not very descriptive in my opinion. It works with the other modules because all those deinitialize functions are scoped in a class, so the class name will always be used at the call site.

> Why not static like in the other modules?

It's a free function.